src/ui/server/constants/supported-currencies.ts

```diff
@@ -4,3 +4,4 @@ export const EUR = 'EUR';
 export const GBP = 'GBP';
 export const JPY = 'JPY';
 export const USD = 'USD';
+export const AED = 'AED';
```

Comment:
Should this be in here? `AED` is not a standard/"supported" currency?

In my PR when testing for a non-standard currency - currently only for an E2E test, have added this to fixtures to be clearer:

```js
export const NON_STANDARD_CURRENCY_CODE = AED_CURRENCY_CODE;
```